src/main/webapp/js/TextEditor.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Initialize the iframe into a RTC
+ * TODO: CSS the frame
+ */
+function init(id) {
+	// check for null param
+	id = id || "";
+	// Gets the document of iframe
+	var doc = document.getElementById(id).contentDocument;
+
+	// Make iframe into a text editor
+	doc.designMode = "on";
+	doc.contentEditable = true;
+	doc
+			.write('<html><head><style>body{ margin:3px; word-wrap:break-word; word-break: break-all; }</style></head><body>CodeUtest</body></html>');
+};
+// TODO: make into a switch statement while adding more functions
+function setFunction(method) {
```

Comment:
This method should probably have some sort of default or error thrown if the string doesn't match one of the case statements.